documentation/modules/rn-2.2.adoc

```diff
@@ -9,21 +9,29 @@ You can migrate virtual machines (VMs) from VMware vSphere or {rhv-full} to {vir
 
 The release notes describe new features and enhancements, known issues, and technical changes.
 
-// [id="technical-changes-22_{context}"]
-// == Technical changes
-//
-// .TBD
-//
-// TBD
-//
-// [id="new-features-and-enhancements-22_{context}"]
-// == New features and enhancements
-//
-// This release adds the following features and improvements.
-//
-// .TBD
-//
-// TBD
+[id="technical-changes-22_{context}"]
+== Technical changes
+
+.Setting the precopy interval for warm migration
+
+You can set the interval between snapshots during the precopy stage of warm migration.
+
+[id="new-features-and-enhancements-22_{context}"]
+== New features and enhancements
+
+This release adds the following features and improvements.
+
+.Downloading logs by using the web console
+
+You can download logs for a migration plan or a migrated VM in the {project-full} web console.
```

Comment:
Change "in the" to "by using the". IBM. Change "{project-full}" to the attribute for MTV, probably project-short.